donkeycar/templates/complete.py

```diff
@@ -137,6 +137,17 @@ def drive(cfg, model_path=None, use_joystick=False, model_type=None, camera_type
             from donkeycar.parts.robohat import RoboHATController
             
             ctr = RoboHATController()
+        elif "custom" == cfg.CONTROLLER_TYPE:
+            #
+            # custom controller created with `donkey createjs` command
+            #
+            from my_joystick import MyJoystickController
```

Comment:
Maybe add some checks if the module import worked, so that people will not get a generic python error but something more obvious with explanation how to fix it?
Some people just don't read docs...